packages/fela/src/generateAnimationName.js

```diff
@@ -1,4 +1,7 @@
 /* @flow */
-export default function generateAnimationName(id: number): string {
-  return `k${id}`
+export default function generateAnimationName(
+  rendererId: string,
```

Comment:
Can we switch position here? I'm not sure if anyone uses that specific function separately, but it'd be nice to break it if the rendererId is missing. Also set a default param value of `''`

Reply:
Updated 👍